app/assets/stylesheets/events.scss

```diff
@@ -0,0 +1,238 @@
+// Place all the styles related to the events controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+@import url("http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900,400italic");
+@import url("//netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css");
+$site-white: #E1E3E3;
+$site-brown: #544D3B;
+$site-black: #262626;
+$site-tan: #A48E74;
+$site-orange: #FF9E43;
+$site-cream: #FFD37C;
+$site-light-grey: #f4f7fb;
+$site-red: #e56c69;
+  
+.hero-unit3 {
+  background-image: image-url("people-fun2.png");
+  background-size: cover;
+  height: 650px;
+  padding-top: -100px;
+}
+  .event-list {
+    list-style: none;
+    font-family: 'Lato', sans-serif;
+    margin: 0px;
+    padding: 0px;
+  }
+
+  .event-list > li {
+    background-color: rgb(255, 255, 255);
+    box-shadow: 0px 0px 5px $site-brown;
+    box-shadow: 0px 0px 5px rgba(51, 51, 51, 0.7);
```

Comment:
Property `box-shadow` already defined on line 31<br>`0px` should be written without units as `0`